contracts/JobRegistry.sol

```diff
@@ -42,12 +44,30 @@ contract JobRegistry {
     /// @param employeeId the talentLayerId of the employee
     /// @param initiatorId the talentLayerId of the user who initialized the job
     /// @param jobDataUri token Id to IPFS URI mapping
+    /// @param status job status
     event JobCreated(
         uint256 id,
         uint256 employerId,
         uint256 employeeId,
         uint256 initiatorId,
-        string jobDataUri
+        string jobDataUri,
+        Status status
+    );
+
+    /// @notice Emitted after a new job is created
+    /// @param id The job ID
+    /// @param employerId the talentLayerId of the employer
+    /// @param employeeId the talentLayerId of the employee
+    /// @param initiatorId the talentLayerId of the user who initialized the job
+    /// @param jobDataUri token Id to IPFS URI mapping
+    /// @param status job status
+    event JobEmployeeAssigned(
+        uint256 id,
+        uint256 employerId,
```

Comment:
we can optim gas here, and don't send in the event data we already know as employerId, initiatorId and jobDataUri

Reply:
OK i delete it